tdrs-backend/tdpservice/users/migrations/0022_add_clam_av_file_scan_perms.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.2.5 on 2021-09-13 20:53
+
+from django.db import migrations
+
+from tdpservice.users.permissions import (
+    create_perms,
+    get_permission_ids_for_model,
+    view_permissions_q
+)
+
+
+def set_clamav_file_scan_permissions(apps, schema_editor):
+    """Set permissions for the ClamAVFileScan model for relevant groups."""
+    ofa_admin = apps.get_model('auth', 'Group').objects.get(name='OFA Admin')
```

Comment:
Only OFA Admin and OFA System Admin will be able to see these results. No one may edit them other than the superuser.